.travis.yml

```diff
@@ -3,7 +3,12 @@ os: osx
 osx_image: xcode10.1
 cache: ccache
 before_install:
- - $TRAVIS_BUILD_DIR/supercollider/.travis/before-install-osx.sh --qt=true
+ # copied from $TRAVIS_BUILD_DIR/supercollider/.travis/before-install-osx.sh:
+ - export HOMEBREW_NO_AUTO_UPDATE=1
+ - export HOMEBREW_NO_INSTALL_CLEANUP=1
+ - brew install libsndfile ccache qt5
+ - export PATH="/usr/local/opt/ccache/libexec:$PATH"
+ - gem install xcpretty
```

Comment:
FYI, in order to use xcpretty you have to pipe the output of the cmake build step into it, see here: https://github.com/supercollider/supercollider/blob/develop/.travis/script-osx.sh#L5-L9

i don't know if you need it here at all since the SC build gets silenced completely with `-quiet`, and the build step for this project uses the makefile generator.

Reply:
ah okay, thanks for catching this! Wasn't sure how xcpretty is used. `-quiet` is my previous addition actually, since the log file was getting too big (and Travis was cancelling the build). I'll remove xcpretty here.